### PR TITLE
Tasnet Segmenter

### PR DESCRIPTION
**Pull request by thequilo**
Uses the Segmenter in the TasNet example. I never used the local shuffle buffer before. Is the buffer size of 128 reasonable?

### CODE REVIEW

**Review by boeddeker — APPROVED** [review]

File: padertorch/contrib/examples/source_separation/tasnet/train.py [hunk 1]
```diff
@@ -316,8 +331,8 @@ def prepare_and_train(_run, _log, trainer, train_dataset, validate_dataset,
 
     # Perform a test run to check if everything works
     trainer.test_run(
-        prepare_iterable_captured(db, train_dataset),
-        prepare_iterable_captured(db, validate_dataset),
+        prepare_dataset_captured(db, train_dataset, shuffle=True),
+        prepare_dataset_captured(db, validate_dataset, shuffle=False),
```
Comment (jensheit) [inline padertorch/contrib/examples/source_separation/tasnet/train.py:335]: I was thinking whether one should deactivate the segmenter during validation

**Review by jensheit — APPROVED** [review]

File: padertorch/contrib/examples/source_separation/tasnet/train.py (hunk 1 above)
Comment (thequilo) [reply padertorch/contrib/examples/source_separation/tasnet/train.py]: It's now deactivated during validation.
